src/scripts/main.js

```diff
@@ -1 +1,12 @@
 'use strict';
+
+const largeImg = document.querySelector('#largeImg');
+const imgList = document.querySelector('#thumbs');
+
+imgList.addEventListener('click', (event) => {
```

Comment:
```suggestion
// eslint-disable-next-line no-shadow
imgList.addEventListener('click', (event) => {
```

Cause event is global object linter shows that annoying message.